src/read/pdb/parser.rs

```diff
@@ -18,9 +19,17 @@ use std::io::{BufRead, BufReader};
 /// # Related
 /// If you want to open a file from memory see [`open_pdb_raw`]. There is also a function to open a mmCIF file directly
```

Comment:
This comment needs updating too, currently it points to a function marked as deprecated